test/test_unary_ufuncs.py

```diff
@@ -129,7 +145,8 @@ def generate_tensors_from_vals(vals, device, dtype, domain):
 # The randomly generated values can be restricted by the domain
```

Comment:
I think it would be nice to document `filter` argument here.

Reply:
done